.editorconfig

```diff
@@ -0,0 +1,15 @@
+# http://editorconfig.org
```

Comment:
we don't need this if we are going to autoformat with black

Reply:
It helps VSCode do the right thing. I tried.